ci/jenkins/scripts/test-eks.sh

```diff
@@ -111,6 +111,8 @@ install_common_packages
 echo "Building Nephe Docker image"
 make build
 
+export AWS_DEFAULT_REGION=${TF_VAR_region}
```

Comment:
at line 60, TF_VAR_Region is set. do you need both TF_VAR and AWS_DEFAULT_REGION. If not, remove one. if yes, probably set the variable at argument parsing.